packages/store-engine/src/main/engine/FileSystemEngine.ts

```diff
@@ -96,7 +96,7 @@ export class FileSystemEngine implements CRUDEngine {
       try {
         data = JSON.stringify(entity);
       } catch (error) {
-        data = entity.toString();
+        data = String(entity);
```

Comment:
Not entirely sure about this one..

Reply:
Since tests don't fail it seems to be ok.